.github/workflows/ci.yaml

```diff
@@ -5,48 +5,50 @@ on:
     branches:
       - master
 env:
-  PYTHON_VERSION: 3.8
+  MIN_PYTHON_VERSION: 3.8
   PY_COLORS: 1
+  IS_MAIN_PYTHON: ${{ matrix.python-version == env.MIN_PYTHON_VERSION && matrix.platform == 'ubuntu-latest' }}
```

Comment:
I like how you removed the duplication by defining it in the environment!

Though, I think it may fail here since `matrix` context does not exist in the global environment. We will need to define this in the job environment instead